config/fcos/v1_0/translate_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2022 Red Hat, Inc
```

Comment:
In the commit message for this commit, it'd still be useful to clarify that we're only touching specs <= 1.2.